cider/common/Stopwatch.h

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ * Copyright (c) 2016-2022 ClickHouse, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#pragma once

```

Comment:
necessary file?

Reply:
It will be used in HashTable.h to record time cost of resize(rehash) when debug is enabled. Anyway, let me remove this since it's our first version and add back later.